ihatemoney/templates/list_bills.html

```diff
@@ -105,7 +105,13 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
             <td>{{ bill.date }}</td>
             <td>{{ bill.payer }}</td>
             <td>{{ bill.what }}</td>
-            <td>{{ bill.owers|join(', ', 'name') }} </td>
+            <td>{% if bill.owers|length == g.project.members|length -%}
```

Comment:
Does this check plays nice with disabled users ?

Reply:
@Glandos, I'm not sure what your :+1: answer means? Is this handled correctly? Thanks :)

Reply:
Ahah, I was sure it was misleading.
No, it means: nice catch. I didn't think about disabled users, so I have to test it. Or at least, it has to be tested.

Reply:
Communication on the internet? Misleading? I have no idea what you're talking about ;-)

Anyway thanks for the help on this. As usual these days (launching a business also has its downsides when it comes to spare time) I probably won't take it over, but if anyone wants, go for it!